README.rst

```diff
@@ -15,6 +15,7 @@ Before starting, you'll need:
  - Docker
  - `docker-compose <https://docs.docker.com/compose/>`_
  - curl
+ - `libsnappy <https://github.com/google/snappy>`_ (`apt-get install libsnappy-dev` / `yum install libsnappy-devel`)
```

Comment:
> To get snappy working, I found I also needed (on a fresh VM):
> 
> ```
> gcc
> gcc-c++
> python3-devel
> ```


Right, if you want to build fastavro from source, you need to be able to compile C code. But you shouldn't need to do that in a modern version of `pip`, since pip should install a wheel for fastavro, which includes the compiled binaries for you. In other words, this will fail:
```
$ docker run --rm -it centos:7 /bin/bash -c "
yum update -y
yum install python3 -y
pip3 install fastavro

[snip]
    building 'fastavro._read' extension
    creating build/temp.linux-x86_64-3.6
    creating build/temp.linux-x86_64-3.6/fastavro
    gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector-strong --param=ssp-buffer-size=4 -grecord-gcc-switches -m64 -mtune=generic -D_GNU_SOURCE -fPIC -fwrapv -fPIC -I/usr/include/python3.6m -c fastavro/_read.c -o build/temp.linux-x86_64-3.6/fastavro/_read.o
    unable to execute 'gcc': No such file or directory
    error: command 'gcc' failed with exit status 1
"
```

But this will succeed:
```
docker run --rm -it centos:7 /bin/bash -c "
yum update -y
yum install python3 -y
pip3 install pip --upgrade
pip install fastavro
"

[snip]
Collecting fastavro
  Downloading fastavro-0.23.4-cp36-cp36m-manylinux2010_x86_64.whl (1.4 MB)
     |################################| 1.4 MB 12.7 MB/s 
Collecting pytz
  Downloading pytz-2020.1-py2.py3-none-any.whl (510 kB)
     |################################| 510 kB 45.1 MB/s 
Installing collected packages: pytz, fastavro
Successfully installed fastavro-0.23.4 pytz-2020.1

```
In general, the pip that comes with OS package managers always needs to be updated before using it.

> Also, the package name for yum is `snappy-devel`

You're totally right, I'll fix that.